.gitignore

```diff
@@ -8,3 +8,9 @@ Cargo.lock
 
 # These are backup files generated by rustfmt
 **/*.rs.bk
+
+qepy.deroff
+pyparseqelectrotech.py
+main.py
+deroff.py
+.qelectrotech.1
```

Comment:
I believe these changes are not required.

Reply:
You're right that was on my local one, oops. db0a659